### PR TITLE
add asking database connection credits on init

### PR DESCRIPTION
**Pull request by mubat**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

**Comment by dmirogin** [comment]
@Mubat Thanks, could you provide more info about this usecase ?
@samdark Could you add labels for pull request like missing usecase and other ?

**Comment by Mubat** [comment]
@dmirogin, changes provide method on *init* script that can ask user to fill parameter values. It useful for `@common/config/main-local.php` file to set database connection credits inline, without navigation to file after project initialization and set these credits. 
After implement this users can set `php yii migrate` command at `post-install-cmd` composer.json section that safe time on project deployment.

File: init [hunk 1]
```diff
@@ -46,6 +46,8 @@ if (empty($params['env']) || $params['env'] === '1') {
     $envName = $params['env'];
```
Comment (cebe) [inline init]: >  init 100755 → 100644 

permission should stay as executeable!

**Comment by mikk150** [comment]
I have created 3 stages of config for that

no suffix(exists already)
-local. suffix(exists already)
-env suffix(does not exist)

environment folder only has -env suffixed  config files
-local suffixed files come from https://github.com/deployphp/deployer

**Comment by samdark** [comment]
While the idea itself is good, we can't merge the pull request in its current state.

**Pull request closed without merging**